docs/uml.py

```diff
@@ -595,3 +603,12 @@ def compile_files_plantuml(input_dir: Path, output_dir: Path, executable: Path)
     """
     command = f'java -jar "{executable}" "{input_dir}" -svg -o "{output_dir}"'
     subprocess.call(shlex.split(command))
+
+
+def test_network_build():
+    """
+    This test is only for debugging purposes.
+    """
```

Comment:
Wäre es dann nicht besser in einem eigenen module/datei besser aufgehoben?
bspw. in `docs/test_uml.py` ?

Reply:
Naja, ob ich für das kleine Ding jetzt nen eigenes module erstellen will ^^ Es ist ja kein Test, der in das CI mit reinspielt, es ist wie gesagt nur für's debuggen

Reply:
Es hätte den Vorteil dass man den Test zum Debugger schon auf der Dateiebene sieht

Reply:
Nur für dich https://github.com/bo4e/BO4E-python/pull/453/commits/425e04f7139aa3526aa9aa7601e90b5f96dbc322 :P